cfn.properties

```diff
@@ -0,0 +1,21 @@
+name=Cloudformation Plugin for SonarQube
+category=External Analysers
+descripton=Cloudformation rules, supporting cfn-nag
+homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
+archivedVersions=1.0.1
+publicVersions=1.0.2
+
+defaults.mavenGroupId=com.hack23.sonar
+defaults.mavenArtifactId=sonar-cloudformation-plugin
+
+1.0.2.description=Update dependency com.fasterxml.jackson.core:jackson-databind to v2.10.0.pr2
```

Comment:
Especially since this description is purely technical, I'd suggest you use instead something like "Initial Marketplace entry"

Reply:
fixed